pkg/provisioningv2/capi/capi.go

```diff
@@ -26,8 +31,13 @@ func init() {
 	_ = clientgoscheme.AddToScheme(schemes.All)
 	_ = clusterv1alpha3.AddToScheme(schemes.All)
 	_ = clusterv1alpha4.AddToScheme(schemes.All)
+	_ = clusterv1beta1.AddToScheme(schemes.All)
 	_ = expv1alpha3.AddToScheme(schemes.All)
+	_ = expv1alpha4.AddToScheme(schemes.All)
```

Comment:
How do we identify we need exp v1alpha4? I looked at `capi-webhooks.yaml` and all entries are using v1beta1.

Reply:
I am not entirely sure how we would verify which schemes we need, especially with all the upgrade scenarios.

Reply:
Oh okay, so we need to add all of them when bumping capi 👍🏼